content/docs/configuration-system/content.md

```diff
@@ -0,0 +1,220 @@
+Another cool thing is that knoxite will recognize encrypted configuration files
+while trying to read them out. Therefore you can reference them just like a
+normal configuration file - without the `crypto://` protocol prefix - for
+further usage.
+
+### Creating an alias
+The `alias` command creates a shorthand for a repository's configuration. In
+this example we're assigning the alias `fort` to the repo located at `/tmp/knox`
+```
+$ knoxite config alias fort -r /tmp/knox
+```
+
+Looking at our configuration again we can see that we created the shorthand
+`fort` with a configuration profile:
+```json
+{
+    "repositories": {
+        "fort": {
+            "url": "/tmp/knox",
+            "compression": "",
+            "tolerance": 0,
+            "encryption": "",
+            "store_excludes": null,
+            "restore_excludes": null
+        }
+    }
+}
+
+```
+
+This shorthand can now further be used in the `-r | --repo` flag while executing
+other commands:
+```
+$ knoxite -r fort store latest .
+...
+```
+
+### Setting values
+The `set` command lets you specify configuration values for an alias stored in
+the knoxite configuration. This is especially useful while working with
+encrypted configuration files.
+
+```
+Usage:
+  knoxite config set <option> <value> [flags]
+```
+
+The option value consists of the repos alias and the configuration value to set
+separated by a dot.
+
+This is how this command can look like in action:
+```
+$ knoxite config set fort.compression gzip
+```
+Here we set `gzip` as the default compression algorithm for the `fort`
+repository which looks like this in the config:
+
+```json
+{
+    "repositories": {
+        "fort": {
+            "url": "/tmp/knox",
+            "compression": "gzip",
+            "tolerance": 0,
+            "encryption": "",
+            "store_excludes": null,
+            "restore_excludes": null
+        }
+    }
+}
+```
+
+The `store_excludes` and `restore_excludes` are defined as an array of strings
+and will work with the pattern syntax specified in Golangs
+`filepath.Match`-Method. You can find it [here](https://golang.org/pkg/path/filepath/#Match).
+
+Here's an example:
+```
+$ knoxite config set fort.store_excludes dont/store/this/folder\* and/this/file
+```
+
+```json
+{
+    "repositories": {
+        "fort": {
+            "url": "/tmp/knox",
+            "compression": "gzip",
+            "tolerance": 0,
+            "encryption": "",
+            "store_excludes": [
+                "dont/store/this/folder*",
+                "and/this/file"
+            ],
+            "restore_excludes": null
+        }
+    }
+}
+```
+
+### Converting between backends
+The `convert` command helps you to translate between different configuration
+backends. There are currently three types of configuration backends supported
+by knoxite:
+- `file://`:   normal unencrypted json file (default)
+- `crypto://`: aes gcm encrypted file
+- `mem://`:    no file - everything in memory
```

Comment:
everything in memory and will be gone after reboot

Reply:
I'm not sure if there are other cases where you could loose a configuration stored in memory

Reply:
I just thought it would be great to explain this more detailled that people who do not know about in memory storage don't lose their configs.